lib/CMakeLists.txt

```diff
@@ -10,13 +10,13 @@ include_directories(
 set(SOURCES
     src/connector/client_metadata.cc
     src/connector/connection.cc
-    src/connector/connector.cc
     src/connector/session_association.cc
```

Comment:
I didn't have a need to. I can definitely do that to make it obvious it's only used for v1.